docs/src/content/docs/commands/QWATCH.md

```diff
@@ -3,81 +3,123 @@ title: QWATCH
 description: The `QWATCH` command is a novel feature designed to provide real-time updates to clients based on changes in underlying data.
 ---
 
-The `QWATCH` command is a novel feature designed to provide real-time updates to clients based on changes in underlying data. It operates similarly to the `SUBSCRIBE` command but focuses on SQL-like queries over data structures. Whenever data modifications affect the query's results, the updated result set is pushed to the subscribed client. This eliminates the need for clients to constantly poll for changes.
+The `QWATCH` command is a novel feature designed to provide real-time updates to clients based on changes in underlying
+data. It operates similarly to the `SUBSCRIBE` command but focuses on SQL-like queries over data structures. Whenever
+data modifications affect the query's results, the updated result set is pushed to the subscribed client. This
+eliminates the need for clients to constantly poll for changes.
 
-This command is what makes DiceDB different from Redis and uniquely positions it as the easiest and most intuitive way to build real-time reactive applications like leaderboards.
+This command is what makes DiceDB different from Redis and uniquely positions it as the easiest and most intuitive way
+to build real-time reactive applications like leaderboards.
 
 ## Protocol Support
 
-| Protocol | Supported |
-| -------- | --------- |
-| TCP-RESP | ✅        |
-| HTTP     | ✅        |
-| WebSocket| ❌        |
+| Protocol  | Supported |
+|-----------|-----------|
+| TCP-RESP  | ✅         |
+| HTTP      | ✅         |
+| WebSocket | ❌         |
 
 ## Syntax
 
-```
-QWATCH dsql-query
+```bash
+QWATCH <dsql-query>
 ```
 
 ## Parameters
 
-- dsql-query: A SQL-like query specifying the data to be monitored and operation to be performed. The query syntax is as follows:
+| Parameter    | Description                                                                         | Type   | Required |
+|--------------|-------------------------------------------------------------------------------------|--------|----------|
+| `dsql-query` | A SQL-like query specifying the data to be monitored and operation to be performed. | String | Yes      |
 
-  - `SELECT`: Specifies the fields to be returned, `$key`, `$value`, and `$value.<attr>`.
-  - `WHERE`: Optional clause for filtering results based on conditions.
-  - `LIKE`: Optional clause within WHERE to specify the key pattern and supports the `%` operator from SQL.
-  - `ORDER BY`: Optional clause for sorting results.
-  - `LIMIT`: Optional clause to limit the number of results.
+## DSQL Query
 
-## Query Syntax
+A SQL-like query specifying the data to be monitored and operation to be performed.
+
+### Syntax
 
 ```sql
-SELECT $key, $value
-WHERE condition
-ORDER BY field [ASC | DESC] LIMIT n
+SELECT $key,
+       $value WHERE condition [AND | OR] $key LIKE 'pattern' [AND | OR] ($key != n [AND | OR] $value > n)
+ORDER BY field [ASC | DESC]
+    LIMIT n
```

Comment:
This isn't the correct grammar for the command. Also, we haven't yet defined a formal grammar, let's stick with the existing syntax:
```sql
SELECT $key, $value
WHERE condition
ORDER BY field [ASC | DESC] LIMIT n
```

Reply:
Reverted!